tox.ini

```diff
@@ -10,7 +10,7 @@ envlist = py{27,36,37,py,py3}{,-ansi2html}, flake8
 commands = pytest -v -r a {posargs}
 deps =
     pytest-xdist
-    pytest-rerunfailures
+    pytest-rerunfailures <= 4.1
```

Comment:
Please add a comment referring to https://github.com/pytest-dev/pytest-rerunfailures/issues/77 so we can remove this in the future. Otherwise, this lgtm.